pkg/operatorsource/configuring.go

```diff
@@ -70,12 +76,43 @@ func (r *configuringReconciler) Reconcile(ctx context.Context, in *marketplace.O
 
 	out = in
 
-	manifests := r.datastore.GetPackageIDsByOperatorSource(in.GetUID())
+	r.logger.Infof("Downloading from [%s]", in.Spec.Endpoint)
+
+	manifests, err := r.downloadManifests(&in.Spec, in.Namespace)
+	if err != nil {
+		nextPhase = phase.GetNextWithMessage(phase.Configuring, err.Error())
+		return
+	}
+
+	if len(manifests) == 0 {
+		err = errors.New("The operator source endpoint returned an empty manifest list")
+
+		// Moving it to 'Failed' phase since human intervention is required to
+		// resolve this situation. As soon as the user pushes new operator
+		// manifest(s) registry sync will detect a new release and will trigger
+		// a new reconciliation.
+		nextPhase = phase.GetNextWithMessage(phase.Failed, err.Error())
+		return
+	}
+
+	r.logger.Infof("%d manifest(s) scheduled for download by the operator-registry pod from the operator source endpoint", len(manifests))
```

Comment:
`%d manifest(s) scheduled for download in the operator-registry pod` will suffice